portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,88 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Prompt client for their name and give them a special greeting.
+ */
+function addSpecialGreeting() {
+    //var client = prompt("What is your name?");
+    fetch('/data').then(response => response.json()).then(greeting => {
+        var box = document.getElementById('special-greeting-container');
+        box.innerText = greeting;
+    })
+}
+
+/**
+ * Adds a random fact to the page
+ */
+function addRandomFact() {
```

Comment:
A lot of these changes seem to be duplicate changes from previous pull requests. Try to submit your pending pull requests first, then merge the changes back to this branch. That way the changes won't show up twice.

Reply:
I've just merged master into this branch and pushed the changed, but the changes persist above. Should I create a new branch to single-out the intended commits after the repeated commits? Or is there some better way to get rid of them?

Reply:
Looks like that took care of most of the differences.